make build error

go: extracting github.com/fatih/structtag v1.1.0
go: extracting golang.org/x/xerrors v0.0.0-20191204190536-9bdfabe68543
go: extracting github.com/kr/text v0.1.0
go: extracting golang.org/x/text v0.3.2
all modules verified
(re)installing /root/.go//bin/promu-v0.5.0
flag provided but not defined: -modfile
usage: go build [-o output] [-i] [build flags] [packages]
Run 'go help build' for details.
make: *** [/root/.go//bin/promu-v0.5.0] Error 2

--- a/README.md
+++ b/README.md
@@ -228,6 +228,8 @@ Prometheus is configured to monitor only the namespace configured in `namespace`
 
 `kind` has cadvisor built in and default Prometheus is set to monitor it.
 
+4. `thanosbench` needs to be compiled with Go 1.14 or later.
+
 ### Benchmarks
 
 * [Remote read](benchmarks/remote-read/README.md)

